src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
```

Comment:
The class Circle should not start with an empty line after the class definition. Please remove the empty line to adhere to good coding practices.